<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignalsExport</class>
 <widget class="QWidget" name="SignalsExport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1054</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_loadFileExportSettings">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>120</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Load file export settings ...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_saveFileExportSettings">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>150</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Save file export settings ...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_exportFiles">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>230</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Export file (s) ...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Export file type: (HVCB)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_newFileExportSettings">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>90</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>New file export settings ...</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>300</y>
     <width>951</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string>Settings preview</string>
   </property>
   <widget class="QPushButton" name="pushButton_signalDownList">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>120</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Down</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Description:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_addSignaToFile">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>30</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Add signal</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_fileHeader">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>231</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Travel, current and arc voltage</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_removeSignalToFile">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>60</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove signal</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>256</width>
      <height>171</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Time [ms]</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Signal 1 - (opis)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Signal 2 - (opis)</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>File header:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>162</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Export nr. phases:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_signalUpList">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>90</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Up</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>202</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Export file type:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_fileDescription">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>231</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Description note</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>160</y>
      <width>110</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="radioButton_1phase">
       <property name="text">
        <string>1ph</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_3phase">
       <property name="text">
        <string>3ph</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>145</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="radioButton_fileTypeInput">
       <property name="text">
        <string>input </string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_fileTypeOutput">
       <property name="text">
        <string>output</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="lineEdit_nazivFileSettingsa">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>231</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>file setting ...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Setting name:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_updateSettings">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>240</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Update settings</string>
    </property>
   </widget>
  </widget>
  <widget class="QTableWidget" name="tableWidget_ExportFileSettings">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>721</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
